### PR TITLE
Replace usage js with WasmJs in README.md

### LINKED ISSUE #256: Adjust readme file to replace kotlin-js with wasm

### PR DESCRIPTION
**Pull request by e5l**
Fix #256

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,34 +6,51 @@
...
-A kotlinx.html library provides DSL to build HTML to [Writer](https://docs.oracle.com/javase/8/docs/api/java/io/Writer.html)/[Appendable](https://docs.oracle.com/javase/8/docs/api/java/lang/Appendable.html) or DOM. Available to all Kotlin Multiplatform targets and browser(or other JavaScript engine) for better [Kotlin programming](https://kotlinlang.org) for Web.
+A kotlinx.html library provides a DSL to build HTML to [Writer](https://docs.oracle.com/javase/8/docs/api/java/io/Writer.html)/[Appendable](https://docs.oracle.com/javase/8/docs/api/java/lang/Appendable.html) or DOM. Available to all Kotlin Multiplatform targets and browser(or other JavaScript engine) for better [Kotlin programming](https://kotlinlang.org) for Web.
 
 # Get started
 
 See [Getting started](https://github.com/kotlin/kotlinx.html/wiki/Getting-started) page for details how to include the library.
 
 # DOM
-You can build DOM tree with JVM and JS naturally
+You can build a DOM tree with JVM, JS, and WASM naturally
```
[the remark this answers is not in the document]
Comment (vnikolova) [reply README.md:16]: Also, what does naturally mean here?

[the remark this answers is not in the document]
Comment (e5l) [reply README.md:16]: I guess it should be like `easy`

File: README.md [hunk 2]
```diff
@@ -6,34 +6,51 @@
 
 # kotlinx.html
 
-A kotlinx.html library provides DSL to build HTML to [Writer](https://docs.oracle.com/javase/8/docs/api/java/io/Writer.html)/[Appendable](https://docs.oracle.com/javase/8/docs/api/java/lang/Appendable.html) or DOM. Available to all Kotlin Multiplatform targets and browser(or other JavaScript engine) for better [Kotlin programming](https://kotlinlang.org) for Web.
+A kotlinx.html library provides a DSL to build HTML to [Writer](https://docs.oracle.com/javase/8/docs/api/java/io/Writer.html)/[Appendable](https://docs.oracle.com/javase/8/docs/api/java/lang/Appendable.html) or DOM. Available to all Kotlin Multiplatform targets and browser(or other JavaScript engine) for better [Kotlin programming](https://kotlinlang.org) for Web.
```
[the remark this answers is not in the document]
Comment (e5l) [reply README.md:9]: thanks, fixed

File: README.md [hunk 3]
```diff
@@ -6,34 +6,51 @@
...
 # Get started
 
 See [Getting started](https://github.com/kotlin/kotlinx.html/wiki/Getting-started) page for details how to include the library.
 
 # DOM
-You can build DOM tree with JVM and JS naturally
+You can build a DOM tree with JVM, JS, and WASM naturally
 
-See example for JavaScript-targeted Kotlin
+See example for WasmJs-targeted Kotlin:
```
[the remark this answers is not in the document]
Comment (e5l) [reply README.md:18]: thanks, looks nice

**Pull request merged**